lib/glific/partners/saas.ex

```diff
@@ -94,6 +96,13 @@ defmodule Glific.Partners.Saas do
   def tax_rates(name \\ "Tides"),
     do: saas_field(name, :tax_rates) |> Map.values()
 
+  @doc """
+  Partner API credentials for Guphsup
+  """
+  @spec gupshup_partner_api(String.t()) :: map()
+  def gupshup_partner_api(name \\ "Gupshup Partner API"),
```

Comment:
Keep the consistency with other fields

```suggestion
  def isv_credentails(name \\ "Gupshup Partner API"),
```

Reply:
made requested changes